BaseApi/serverless.yml

```diff
@@ -22,7 +22,7 @@ provider:
 
 package:
 # TODO: Rename zipfile in build.sh and build.cmd to match this
-  artifact: ./bin/release/net6.0/base-api.zip
```

Comment:
Curious. Is this not needed anymore?

Reply:
That's a good question. It looks like the path is arbitrary – it just needs to be consistent in  `build.cmd`/`build.sh` and `serverless.yml`, so rather than having the .NET version number (and therefore having to change this with each upgrade) I _think_ it's safe to make it version-agnostic.

That said, I'm not that familiar with .NET on lambda, so if you think this will break things I'll use the old format.

Reply:
I got no idea, never tried changing the path to exclude the version number. :thinking:

Reply:
For posterity: we decided in Slack to test this in https://github.com/LBHackney-IT/single-view-api so we can be confident the path change won't break anything. Once we know for sure we'll resolve this conversation one way or the other.

Reply:
This as it is built as part of the deplyment process.  I think this is where Serverless looks for the package?  Agreed it is arbitrary.  Dotnet defaults to this with their own version numbers.

Reply:
Late last week we checked Single View API and the change deployed successfully without the version number in the build path (as expected, but it's nice to be sure 🤗 ).